keps/sig-api-machinery/2161-apiserver-default-labels/README.md

```diff
@@ -136,6 +135,7 @@ able to select a single namespace to enforce those controllers. Similarly, it is
 users to remember the names of the namespaces rather than labels associated with those namespaces while defining
 NetworkPolicies or writing admission controller configurations.
 
+Thus, regardless of wether such a user story is valid, it is clearly 'simpler' to implement (as are other similar stories) when a default mechanism for namespace selection is available.
```

Comment:
I think validity matters.  Things that aren't recommended don't need to be easy.